chainerio/cache/file_cache.py

```diff
@@ -77,21 +77,33 @@ def unlock(self):
 
 
 class FileCache(cache.Cache):
-    '''Stores cache data in local filesystem
+    '''Cache system with local filesystem
 
-    Stores cache data in local filesystem,
+    Stores cached data in local filesystem,
```

Comment:
"Cache data on local directory, "
As ``~/.chainer/chainerio/cache`` is not a filesystem